File: phase1/python/day01_shopping_bill.py
# Step 1 - take inputs for 3 items
price1, qty1 = float(input("Item 1 price: ")), int(input("Item 1 quantity: "))
price2, qty2 = float(input("Item 2 price: ")), int(input("Item 2 quantity: "))
price3, qty3 = float(input("Item 3 price: ")), int(input("Item 3 quantity: "))

# Step 2 - calculate each item total
item1_total = price1 * qty1
item2_total = price2 * qty2
item3_total = price3 * qty3

# Step 3 - add them all
bill_total = item1_total + item2_total + item3_total

# Step 4 - check if total > 1000, apply 10% discount
if bill_total > 1000:
    discount = bill_total * 0.1
    final = bill_total - discount
    print(f"Discount applied: ₹{discount:.2f}")
else:
    final = bill_total

# Step 5 - print final amount
print(f"Total Bill: ₹{bill_total:.2f}")
print(f"Final Amount: ₹{final:.2f}")

File: phase1/cpp/day01_shopping_bill.cpp
#include <iostream>
using namespace std;

int main() {
    float price1, price2, price3;
    int qty1, qty2, qty3;

    // Step 1 - inputs
    cout << "Item 1 price: "; cin >> price1;
    cout << "Item 1 quantity: "; cin >> qty1;
    cout << "Item 2 price: "; cin >> price2;
    cout << "Item 2 quantity: "; cin >> qty2;
    cout << "Item 3 price: "; cin >> price3;
    cout << "Item 3 quantity: "; cin >> qty3;

    // Step 2 - item totals
    float item1_total = price1 * qty1;
    float item2_total = price2 * qty2;
    float item3_total = price3 * qty3;

    // Step 3 - bill total
    float bill_total = item1_total + item2_total + item3_total;

    // Step 4 - discount
    float final_amount;
    if (bill_total > 1000) {
        float discount = bill_total * 0.1;
        final_amount = bill_total - discount;
        cout << "Discount applied: Rs." << discount << endl;
    } else {
        final_amount = bill_total;
    }

    // Step 5 - print
    cout << "Total Bill: Rs." << bill_total << endl;
    cout << "Final Amount: Rs." << final_amount << endl;

    return 0;
}